files/scripts/sonic-sync-mode

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
+import argparse
+from swsscommon import swsscommon
+
+def main():
+    parser=argparse.ArgumentParser(description=
+          'Configure enable or disable synchronous mode between orchagent and syncd',
+          formatter_class=argparse.RawTextHelpFormatter,
+          epilog=
+'''
+**sudo** needed for configuring synchronous mode
```

Comment:
>**sudo** [](start = 0, length = 8)

Do you mean bold font? It's difficult to read.

Reply:
Fixed this and moved the CLI to sonic-utilities repo.